### PR TITLE
Enable Contribute page to respond to screen size changes

### PR DESCRIPTION
**Pull request by andrewiggins**
This change enables overflow on the code examples in the contribute page so the content can respond to screen width changes. First step to enabling a responsive mobile experience for the OpenUI website. In a future PR I'll look at adding a very basic accessible mobile menu.

Also fixed JSON5 syntax highlighting.

I based this on some of the suggestions by @aardrian in #170

Before:
image
Note horizontal scrollbar on page, no syntax highlighting

After:
image
Note horizontal scrollbar on code example and not entire page, proper syntax highlighting

Related #170

### CODE REVIEW

File: research/src/pages/contribute.mdx [hunk 1]
```diff
@@ -81,7 +81,7 @@ Add the `$schema` key pointing to our `design-system.schema.json5` schema and co
 {
   $schema: '../schemas/design-system.schema.json5',
   name: 'Ant Design',
-  description: 'A design system with values of Nature and Determinacy for better user experience of enterprise applications.',
+  description: 'An enterprise-class UI design language and React UI library.',
```
Comment (andrewiggins) [inline research/src/pages/contribute.mdx:84]: I've updated this description with the tagline from the Ant Design Readme. I liked it cuz it's a little shorter and reduces the screen size at which as scrollbar is displayed for this code sample

File: research/src/pages/contribute.mdx [hunk 2]
```diff
@@ -198,7 +198,7 @@ The `pathToProposal` key tells your research page where its proposal page is hos
 
 Example:
 
-```
+```markdown
```
Comment (andrewiggins) [inline research/src/pages/contribute.mdx:201]: Currently Prism syntax highlighting doesn't do much with this sample, but the next version of Prism will have first class support for highlighting markdown front matter yaml syntax (PrismJS/prism#2634)

**Comment by yoavweiss** [comment]
@andrewiggins - can you join the wicg to appease the IPR bots?

**Comment by andrewiggins** [comment]
Yup - I've created my W3C account and linked my GitHub account. I've submitted requests to join the WICG and OpenUI Community Groups and am just awaiting approval from my Advisory Committee Rep

**Comment by andrewiggins** [comment]
Rep just approved and the `ipr` check is now passing 🎉

**Comment by yoavweiss** [comment]
@andrewiggins - welcome! :)

**Comment by una** [comment]
Nice, thanks for this PR

**Review by una — APPROVED** [review]

**Review by gregwhitworth — APPROVED** [review]
Thank you so much for this PR - and a mobile menu PR would be epic!!!

**Pull request merged**